spark-jobs/pom.xml

```diff
@@ -191,6 +190,20 @@
                             <exclude>net.jpountz.lz4:*</exclude>
                         </excludes>
                     </artifactSet>
+                    <relocations>
+                        <!-- This was introduced in #86 Set up Standardization and Conformance Menas auth.
```

Comment:
Question: I like comments. Just wonder, if this is not too detailed explanation?

Reply:
I don't mind removing it. It just took some time to find out why we are doing this relocation. I'm slightly inclined to keep it for the future maintainers but will remove it if you think this is too much 😄

Reply:
I definitely like that it's explained why the relocation was added. Just found it untypical, compared to what we usually have, that it got down to the level of mentioning the exception class. Fine if it stays, we can always shorten the comment if future. in case it's found to be long. 😉